resources/script.js

```diff
@@ -5,6 +5,23 @@ function currentPosition() {
   return parseInt(document.querySelector('.slide:not(.hidden)').id.slice(6));
 }
 
+/**
+ * If passed to navigate, will move to the first slide, independent of the
+ * current location, rather than a relative amount.
+ */
+var FIRST_SLIDE = -9999999;
+
+/**
+ * If passed to navigate, will move to the last slide, independent of the
+ * current location, rather than a relative amount.
+ */
+var LAST_SLIDE = 9999999;
+
+/**
+ * Whether to allow looping from the last back to the first or backwards from
+ * the first to the last.
+ */
+var ALLOW_LOOPING = true;
 
 /**
  * Navigates forward n pages
```

Comment:
Yeah, since it uses relative values, "1" means "advance one slide", so had to pick values outside any useful range. I suppose if someone ends up with a 10,000,000 slide presentation there will be problems with this feature, but it seems like that's a fair "bug" to leave latent.